src/PKSim.Core/CoreConstants.cs

```diff
@@ -707,8 +707,6 @@ public static class ORM
          public static readonly string ViewMoleculeStartFormulas = "VIEW_MOLECULE_START_FORMULAS";
          public static readonly string ViewOntogenies = "VIEW_ONTOGENIES";
          public static readonly string ViewCategory = "VIEW_CATEGORIES";
-         public static readonly string ViewOldParameterNameMappings = "VIEW_OLD_PARAMETER_NAME_MAPPING";
-         public static readonly string ViewOldCalculationMethodNameMappings = "VIEW_OLD_CALCULATION_METHOD_NAME_MAPPING";
          public static readonly string ViewCalculationMethodParameterRates = "VIEW_CALCULATION_METHOD_PARAMETER_RATES";
```

Comment:
obsolete views

Reply:
Next time let's not REMOVE stuff that is not used in hot fix patches. This is going to make merging much harder that required (especially removing files etc..)